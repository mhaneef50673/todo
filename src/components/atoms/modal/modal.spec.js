import React from 'react';
import { mount } from 'enzyme';
import { mountToJson} from 'enzyme-to-json';
import Modal from './index';

describe('Modal Component', () => {
  it('should render the component', () => {
    const tree = mount(
      <Modal title="Test Title" isVisible />
    )
    expect(mountToJson(tree)).toMatchSnapshot();
  });
});