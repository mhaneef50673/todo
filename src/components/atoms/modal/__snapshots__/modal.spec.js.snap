// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Modal Component should render the component 1`] = `
<Modal
  isVisible={true}
  title="Test Title"
>
  <div
    className="modal"
    id="modal"
  >
    <h4>
      Test Title
    </h4>
    <div
      className="content"
    >
      <div />
    </div>
  </div>
</Modal>
`;
